.github/workflows/conformance.yml

```diff
@@ -16,9 +16,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
+
       - uses: actions/setup-python@5ccb29d8773c3f3f653e1705f474dfaa8a06a912
+        with:
+          python-version: "3.x"
```

Comment:
`3.x` should match the latest stable version of Python 3 available to the action, which in this case should be 3.11. So technically I should make this `>= 3.7`, but the action would fail loudly anyways due to the metadata having the wrong version for the running Python version.